iroh/bench/src/iroh.rs

```diff
@@ -125,18 +128,18 @@ pub async fn connect_client(
 pub fn transport_config(max_streams: usize, initial_mtu: u16) -> QuicTransportConfig {
     // High stream windows are chosen because the amount of concurrent streams
     // is configurable as a parameter.
-    let mut config = QuicTransportConfig::default();
-    config.max_concurrent_uni_streams(max_streams.try_into().unwrap());
-    config.initial_mtu(initial_mtu);
+    let mut builder = QuicTransportConfigBuilder::default();
```

Comment:
maybe have `QuicTransportConfig::builder()` be a thing, requires one less type to import for users